RichClipboard.package/RCClippingsList.class/instance/selectedClippingIndex..st

```diff
@@ -1,4 +1,5 @@
 accessing
 selectedClippingIndex: anIndex
 	"At this moment we do not want any item to be selected"
-	self changed: #selectedClippingIndex
\ No newline at end of file
+	self changed: #selectedClippingIndex.
+	self copyBackToClipboard: anIndex.
```

Comment:
Wir müssen noch überprüfen, 

- ob anIndex = 0 ist -> Dann sollte nichts getan werden (kein selektiertes Item); Im Moment wird ein "OutOfBounds" Error geworfen
- ob selectedClippingIndex wegen einer Tastatureingabe oder wegen eines Klicks ausgeführt wird (im Moment wird auch bei Tastatureingaben ein Clipping kopiert. Durch das entfernen ist zudem im Moment die Suchfunktionalität beeinträchtigt)

Reply:
Das zweite Problem können wir unter Umständen lösen, indem wir in RCPluggableListMorph speichern, ob das letzte Event ein Maus oder Tastaturevent war und das dann ClippingsList mitteilen. Und dann führen wir diese Logik nur aus, wenn es ein Mausevent war.